python/src/spark_rapids_ml/params.py

```diff
@@ -145,6 +145,9 @@ def _param_mapping(cls) -> Dict[str, Optional[str]]:
         - empty string, if a defined Spark Param should just be silently ignored, or
         - None, if a defined Spark Param should raise an error.
 
+        If the class has no Spark ML equivalent, the cuML parameter names should be mapped to themselves,
+        i.e., `{"cuml_param": "cuml_param"}`.
```

Comment:
Seems we do not need this. Existing code directly sets cuml param if the param name matches, as shown here: https://github.com/NVIDIA/spark-rapids-ml/blob/f0fa39cde77e8c6b4b8f598e6740cc1b7579ec06/python/src/spark_rapids_ml/params.py#L342

Reply:
Resolving after our discussion online.